test/cases/underscore.zig

```diff
@@ -0,0 +1,34 @@
+const std = @import("std");
+const assert = std.debug.assert;
+
+test "ignore lval with underscore" {
+    _ = false;
+}
+
+test "ignore lval with underscore (for loop)" {
+    for ([]void{}) |_, i| {
+        for ([]void{}) |_, j| {
+            break;
+        }
+        break;
+    }
+}
+
+test "ignore lval with underscore (while loop)" {
+    while (optionalReturn()) |_| {
+      while (optionalReturn()) |_| {
+          break;
+      }
+//      else |_| {
```

Comment:
looks like you intended to add another test case where you call an `errorableReturn()` with an `else |_|` ?

Reply:
Yes, its WIP.